diff -urN wml-2.0.3.org/Makefile.in wml-2.0.3/Makefile.in
--- wml-2.0.3.org/Makefile.in	Thu Sep  7 21:27:11 2000
+++ wml-2.0.3/Makefile.in	Thu Sep  7 22:19:07 2000
@@ -23,12 +23,12 @@
 FIXPERM         = $(SHTOOL) fixperm
 TARBALL         = $(SHTOOL) tarball -t
 
-prefix          = @prefix@
+prefix          = $(DESTDIR)@prefix@
 exec_prefix     = $(prefix)
-bindir          = $(prefix)/bin
+bindir          = $(DESTDIR)@bindir@
 libsubdir       = @libsubdir@
 libdir          = $(prefix)/lib$(libsubdir)
-mandir          = $(prefix)/man
+mandir          = $(DESTDIR)@mandir@
 catdir          = @catdir@
 tmpdir          = @tmpdir@
 
diff -urN wml-2.0.3.org/wml_aux/iselect/Makefile.in wml-2.0.3/wml_aux/iselect/Makefile.in
--- wml-2.0.3.org/wml_aux/iselect/Makefile.in	Thu Sep  7 21:27:09 2000
+++ wml-2.0.3/wml_aux/iselect/Makefile.in	Thu Sep  7 22:06:32 2000
@@ -26,10 +26,10 @@
 FIXPERM         = $(SHTOOL) fixperm
 
 #   installation paths
-prefix          = @prefix@
-exec_prefix     = @exec_prefix@
-bindir          = @bindir@
-mandir          = ${exec_prefix}/man
+prefix          = $(DESTDIR)@prefix@
+exec_prefix     = $(DESTDIR)@exec_prefix@
+bindir          = $(DESTDIR)@bindir@
+mandir          = $(DESTDIR)@mandir@
 tmpdir          = @tmpdir@
 
 OBJS = \
diff -urN wml-2.0.3.org/wml_backend/p3_eperl/Makefile.in wml-2.0.3/wml_backend/p3_eperl/Makefile.in
--- wml-2.0.3.org/wml_backend/p3_eperl/Makefile.in	Thu Sep  7 21:27:10 2000
+++ wml-2.0.3/wml_backend/p3_eperl/Makefile.in	Thu Sep  7 22:07:32 2000
@@ -45,12 +45,12 @@
 CP              = cp
 
 #   installation paths
-prefix          = @prefix@
+prefix          = $(DESTDIR)@prefix@
 exec_prefix     = $(prefix)
-bindir          = $(prefix)/bin
+bindir          = $(DESTDIR)@bindir@
 libsubdir       = @libsubdir@
 libdir          = $(prefix)/lib$(libsubdir)
-mandir          = $(prefix)/man/man1
+mandir          = $(DESTDIR)@mandir@
 tmpdir          = @tmpdir@
 
 
diff -urN wml-2.0.3.org/wml_common/Makefile.in wml-2.0.3/wml_common/Makefile.in
--- wml-2.0.3.org/wml_common/Makefile.in	Thu Sep  7 21:27:08 2000
+++ wml-2.0.3/wml_common/Makefile.in	Thu Sep  7 22:08:18 2000
@@ -17,12 +17,12 @@
 INSTALL_PROGRAM = $(top_srcdir)/etc/shtool install -c -m 755 -s
 INSTALL_DATA    = $(top_srcdir)/etc/shtool install -c -m 644
 
-prefix          = @prefix@
+prefix          = $(DESTDIR)@prefix@
 exec_prefix     = $(prefix)
-bindir          = $(prefix)/bin
+bindir          = $(DESTDIR)@bindir@
 libsubdir       = @libsubdir@
 libdir          = $(prefix)/lib$(libsubdir)
-mandir          = $(prefix)/man
+mandir          = $(DESTDIR)@mandir@
 
 PATH_PERL       = @PATH_PERL@
 MM_INSTALL_OPTS = PREFIX=$(libdir)/perl \
diff -urN wml-2.0.3.org/wml_include/Makefile.in wml-2.0.3/wml_include/Makefile.in
--- wml-2.0.3.org/wml_include/Makefile.in	Thu Sep  7 21:27:11 2000
+++ wml-2.0.3/wml_include/Makefile.in	Thu Sep  7 22:09:06 2000
@@ -19,12 +19,12 @@
 INSTALL_DATA    = $(top_srcdir)/etc/shtool install -c -m 644
 MKDIR           = $(top_srcdir)/etc/shtool mkdir -f -p -m 755
 
-prefix          = @prefix@
+prefix          = $(DESTDIR)@prefix@
 exec_prefix     = $(prefix)
-bindir          = $(prefix)/bin
+bindir          = $(DESTDIR)@bindir@
 libsubdir       = @libsubdir@
 libdir          = $(prefix)/lib$(libsubdir)
-mandir          = $(prefix)/man
+mandir          = $(DESTDIR)@mandir@
 
 INSTALLPRIVLIB  = @INSTALLPRIVLIB@
 INSTALLARCHLIB  = @INSTALLARCHLIB@
diff -urN wml-2.0.3.org/wml_misc/Makefile.in wml-2.0.3/wml_misc/Makefile.in
--- wml-2.0.3.org/wml_misc/Makefile.in	Thu Sep  7 21:27:09 2000
+++ wml-2.0.3/wml_misc/Makefile.in	Thu Sep  7 22:09:42 2000
@@ -19,12 +19,12 @@
 INSTALL_DATA    = $(top_srcdir)/etc/shtool install -c -m 644
 MKDIR           = $(top_srcdir)/etc/shtool mkdir -f -p -m 755
 
-prefix          = @prefix@
+prefix          = $(DESTDIR)@prefix@
 exec_prefix     = $(prefix)
-bindir          = $(prefix)/bin
+bindir          = $(DESTDIR)@bindir@
 libsubdir       = @libsubdir@
 libdir          = $(prefix)/lib$(libsubdir)
-mandir          = $(prefix)/man
+mandir          = $(DESTDIR)@mandir@
 
 # ------------------------------------------------
 #   TARGETS
