diff -urN wml-2.0.3.org/wml_backend/p3_eperl/eperl_perl5.h wml-2.0.3/wml_backend/p3_eperl/eperl_perl5.h
--- wml-2.0.3.org/wml_backend/p3_eperl/eperl_perl5.h	Thu Sep  7 21:27:10 2000
+++ wml-2.0.3/wml_backend/p3_eperl/eperl_perl5.h	Thu Sep  7 21:38:10 2000
@@ -74,8 +74,10 @@
 #  define PERL_IO_LAYER_ID "Raw/StdIO"
 #endif
 
+#ifndef PL_curstash
 #if (PERL_VERSION < 4) || ((PERL_VERSION == 4) && (PERL_SUBVERSION <= 5))
 #  define PL_curstash curstash
+#endif
 #endif
 
 #ifndef WITH_THR
